src/test/globalSetup.test.ts

```diff
@@ -58,7 +55,8 @@ beforeEach(function () {
     testLogger = setupTestLogger()
     globals.templateRegistry = new CloudFormationTemplateRegistry()
     globals.codelensRootRegistry = new CodelensRootRegistry()
-    globals.telemetry = initTelemetry()
```

Comment:
The reason for calling `initTelemetry` every time was because telemetry is stateful. We don't need to re-instantiate the entire system, but we should at least make sure the state is fully cleared before every test.

Reply:
added clear() calls here.